### ISSUE TITLE
Strong Sparse Reconstruction and Training, but Poor Final Results

### ISSUE THREAD

**Issue by cymerus** [inuex35/ind-bermuda-opensfm#2]
Thank you for the exceptional work on this project, it has been incredibly valuable.

I am currently working on reconstructing scenes from panorama images and have been using OpenSFM for sparse reconstruction and camera parameter estimation. These components are performing remarkably well, and the training process also shows promising results. Specifically:

The training loss consistently decreases throughout, and the final PSNR metrics appear excellent.
Rendered views from the training data also look visually accurate.
However, I am encountering some challenges, despite experimenting with a wide range of parameters. I have attached the training command with the parameters I’ve used, along with screenshots of the resulting scene for your reference.

Could you please help me understand what might be causing these issues? 
Any guidance would be greatly appreciated.

python train.py     --source_path data/scene_1    
                                           --model_path output/scene_1
                                           --panorama     
                                           --iterations 40000     
                                           --position_lr_init 0.0002     
                                           --position_lr_final 0.000002 
                                           --feature_lr 0.0025     
                                           --opacity_lr 0.05     
                                           --scaling_lr 0.005     
                                           --rotation_lr 0.001     
                                           --densify_from_iter 500     
                                           --densify_until_iter 30000     
                                           --densify_grad_threshold 0.00005     
                                           --densification_interval 50     
                                           --opacity_reset_interval 3000     
                                           --lambda_dssim 0.3     
                                           --sh_degree 3 
                                           
                                           
Here are the training metrics:

`Evaluating train: L1 0.016351059079170227 PSNR 29.32947463989258`

And screenshots from resulting reconstruction:
<img width="1195" alt="Screenshot 2024-11-19 at 8 50 04 AM" src="https://github.com/user-attachments/assets/90688c93-afdf-4aab-90ec-01a695b991fe">
